### PR TITLE
ADEN-6046 JWPlayer + Kikimora tracking

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.events.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+	}
+
+	function getNewState() {
+		return {
+			ad: getDefaultState(),
+			video: getDefaultState()
+		}
+	}
+
+	function handleTime(prefix, data) {
```
Comment (WikiaTech) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.events.js:31]: MAJOR Function has a complexity of 11 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [porvata.js#L242](https://github.com/Wikia/app/blob/b44baf349618ebdbd74a61ce2581691caa3d264a/extensions/wikia/AdEngine/js/template/porvata.js#L242): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [porvata.js#L242](https://github.com/Wikia/app/blob/fcef7b4ea598376bad8cea48cde64d5d37129b3b/extensions/wikia/AdEngine/js/template/porvata.js#L242): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads-tracking.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	'ext.wikia.adEngine.video.player.jwplayer.jwplayerTracker'
+], function (eventDispatcher, tracker) {
+	function updateParams(params, currentAd) {
+		var wrapperCreativeId,
+			wrapperId;
+
+		if (currentAd) {
+			params.lineItemId = currentAd.getAdId();
+			params.creativeId = currentAd.getCreativeId();
+			params.contentType = currentAd.getContentType();
```
Comment (kvas-damian) [inline extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads-tracking.js:18]: What do you think about emptying those fileds on `adError`?

Comment (fraszczakszymon) [reply extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads-tracking.js:18]: Done

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [porvata.js#L242](https://github.com/Wikia/app/blob/d1dc287ad20ce43a77e5a72f8a0d0d116ef99371/extensions/wikia/AdEngine/js/template/porvata.js#L242): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by kvas-damian — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [porvata.js#L242](https://github.com/Wikia/app/blob/0abd50033428e65645d14dcfaf4cb12ad64c0c43/extensions/wikia/AdEngine/js/template/porvata.js#L242): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [porvata.js#L242](https://github.com/Wikia/app/blob/d2591864a4aea9c20c71842cce62c5c47d483711/extensions/wikia/AdEngine/js/template/porvata.js#L242): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [porvata.js#L242](https://github.com/Wikia/app/blob/a45d6f27444e9ed872ad4939242848e7b5396bc2/extensions/wikia/AdEngine/js/template/porvata.js#L242): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**